app/views/books/_card.html.erb

```diff
@@ -1,21 +1,29 @@
-<% books_data.each do |book|%>
+<% books_data.each do |book| %>
```

Comment:
I think this will be a bit confusing for me, the partial says that it is card in the name but it prints list of cards. I think having a partial to print one card would be helpful in multiple scenarios than list of cards.

Can we make this partial only about one card ?

Reply:
Yes, that be would be good. I will change it.